test/central-config-disabled.test.js

```diff
@@ -24,18 +25,15 @@ test('central config disabled', function (t) {
       serviceName: 'test',
       captureExceptions: false,
       metricsInterval: 0,
+      apmServerVersion: '8.0.0',
       centralConfig: false
     })
 
     setTimeout(function () {
       t.pass('should not poll APM Server for config')
-      t.end()
       agent.destroy()
       server.close()
+      t.end()
     }, 1000)
   })
-
-  t.on('end', function () {
-    if (existingValue) process.env.ELASTIC_APM_CENTRAL_CONFIG = existingValue
-  })
```

Comment:
Reviewer note: Since the time this test file was written, test files are now always run in a separate process. That means there is no point in restoring envvars for the next test file.